demo/index.ts

```diff
@@ -221,6 +229,17 @@ const createEditor = (server: CollabServer) => {
     required: false,
   });
 
+  createElementButton("Add callout element", calloutElementName, {
+    weighting: "",
+    sourceUrl: "",
+    embedCode: "",
+    caption: "",
+    altText: "",
+    required: false,
+    html:
+      '<div data-callout-tagname="callout-brexit"><h2>Callout<h2><p>callout-brexit</p></div>',
```

Comment:
We should be able to only include the fields we know Composer will provide, giving us a more accurate reflection of production.

Reply:
Good point, have done so now.